packages/react-docsite-components/src/components/ExampleCard/ExampleCard.tsx

```diff
@@ -228,15 +212,13 @@ export class ExampleCardBase extends React.Component<IExampleCardProps, IExample
           </div>
         );
 
-        if (activeCustomizations) {
+        if (activeTheme) {
+          const themeToApply = activeTheme.schemes?.[scheme] || activeTheme;
+
           return (
-            <CustomizerContext.Provider value={{ customizations: { settings: {}, scopedSettings: {} } }}>
-              <Customizer {...activeCustomizations}>
-                <ThemeProvider scheme={_schemes[schemeIndex]}>
-                  <Stack styles={regionStyles}>{content}</Stack>
-                </ThemeProvider>
-              </Customizer>
-            </CustomizerContext.Provider>
+            <ThemeProvider theme={themeToApply}>
```

Comment:
I think the idea of this section was:
1. Clear ALL existing page-wide customizations/theme
2. Merge and apply the local theme and scheme customizations (maybe? a little fuzzy on this part)

Is that still what it does?